uberduck_ml_dev/data/parse.py

```diff
@@ -0,0 +1,163 @@
+# AUTOGENERATED! DO NOT EDIT! File to edit: nbs/data.parse.ipynb (unless otherwise specified).
+
+__all__ = ['CACHE_LOCATION', 'STANDARD_MULTISPEAKER', 'STANDARD_SINGLESPEAKER', 'VCTK']
+
+# Cell
+import os
+import numpy as np
+import sqlite3
+import uuid
+from pathlib import PosixPath
+from tqdm import tqdm
+from pathlib import Path
+import os
+import pandas as pd
+import json
+
+CACHE_LOCATION = Path.home() / Path(".cache/uberduck/uberduck-ml-exp.db")
+STANDARD_MULTISPEAKER = "standard-multispeaker"
+STANDARD_SINGLESPEAKER = "standard-singlespeaker"
+VCTK = "vctk"
+
+
+def _log_filelists(folder, fmt, conn, dataset_name: str = None):
```

Comment:
I don't think `_log_...` is the right verb for this function. Log means "write into an immutable record for observability". How about `_store_filelists` or `_cache_filelists` ?